### ISSUE TITLE
[bug] Increasing number of `processBlock` errors on devnet-user

### ISSUE THREAD

**Issue by gmasgras** [filecoin-project/go-filecoin#2287]
**Describe the bug**
Infra has been seeing a lot of these errors on the usernet lately, 2200 in the past 24hrs

```
processBlock(): processing block from network: blocks state root does not match computed result
```

```
processBlock(): processing block from network: context deadline exceeded
```

**Logs**
https://search-filecoin-logs-feyxjodkwak6pulqamqtqwgddi.us-east-1.es.amazonaws.com/_plugin/kibana/goto/c6316d10244033ea98e8d464f48d942e

**Screenshots**
<img width="869" alt="Screen Shot 2019-03-14 at 3 34 23 PM" src="https://user-images.githubusercontent.com/42549113/54396001-b051ab80-466e-11e9-9873-f78bdf150bdd.png">

**Comment by anorth** [comment]
Am I correct in thinking that any random old/broken/malicious node could broadcast an invalid block and trigger this? If so, it's likely to just be some nodes that are running different code to others.

**Comment by gmasgras** [comment]
> Am I correct in thinking that any random old/broken/malicious node could broadcast an invalid block and trigger this? If so, it's likely to just be some nodes that are running different code to others.

These errors are being thrown by the filecoin nodes that we are running.

**Comment by anorth** [comment]
Right, but are other nodes sending them bad blocks?

**Comment by acruikshank** [comment]
This seems likely to be people mining blocks with the wrong build. It may be corrected when #2262 lands in devnet.

**Comment by gmasgras** [comment]
Seems to be fixed after the `0.1.4` user devnet deploy.
